ARExperiment/ARExperiment/SizeComparisonViewController.swift

```diff
@@ -3,17 +3,61 @@ import SceneKit
 import ARKit
 
 class SizeComparisonViewController: UIViewController, ARExperimentSessionHandler {
+
+    enum CubeModel {
```

Comment:
Can we remove this from the controller? There are is a Models folder if you pull from master and it would make this clearer if it is in another file. 

Also instead of enums we try to use more `static let` and `struct` you can check why in this discussion -> https://github.com/novoda/novoda-autolayout-constraints/pull/3 

So you would get something like this: 
``` 
struct CubeModel {
     let fileName: String 
     let textFileName: String 
}

extension CubeModel {
      static let big = CubeModel(fileName: "cube_5", textFileName: "text_5")
}
```
And this way if we add a new cube or remove we do not need to update the entire switch structure, it makes for easier refactoring and escalation.

Reply:
Ok nice approach 👍